extractors/bookseries/karelians/extractors/kalastus_extractor.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+import core.utils.regex_utils as regexUtils
+from core.pipeline_construction.base_extractor import BaseExtractor
+
+
+class KalastusExtractor(BaseExtractor):
+    extraction_key = 'kalastus'
+
+    def __init__(self, cursor_location_depends_on=None, options=None):
+        super(KalastusExtractor, self).__init__(cursor_location_depends_on, options)
+        self.KALASTUS_PATTERN = r"(?P<kalastus>kalastus)|(kalastamista)|(kalastaa)|(kalastaminen)"
+        self.KALASTUS_OPTIONS = (re.UNICODE | re.IGNORECASE)
+
+    def _extract(self, entry, extraction_results, extraction_metadata):
+        own_kalastus = self._find_kalastus(entry["text"])

```

Comment:
Not entirely sure what this `own_kalastus` variable name is supposed to mean :sweat_smile: